resources/lib/playerinfo.py

```diff
@@ -22,12 +22,14 @@ def __init__(self):
         self.last_pos = None
         self.listen = False
         self.paused = False
-        self.total = 900
+        self.total = 999
```

Comment:
I wondered why this defaulted to 900, and now to 999.
Wouldn't it be better to take something very large, or -1 or 0 instead?

I don't know exactly how this impacts resumepoints and Kodi (does this value gets stored?).

Reply:
You added `push_position` in `onPlayBackStarted` but at that time, there is no video info yet and `xbmc.Player().getTotalTime()` returns 0.0
So, the intention was to reset the resumepoint to "unwatched" and sending position 0.0 and total 999 to VRT NU. I'm not sure what is the best choice here.

Sending real values can also be done, but only at a later time in `onAVStarted`.

Reply:
I did not realize.

If I remember correctly, if you would tag an episode as Watch Later, VRT NU would automatically set resumetime to 0 and totaltime to 100. Which is odd, but at least that is what the VRT NU website is doing.

Reply:
Okay, I'll change it to what VRT NU does.

Reply:
Leave it at None. We set it to 100 when we create an entry in ResumePoints.update() already.
At least that way we do not override a better value with 100.